benchmarks/datasets/join/create_index/pg_search.sql

```diff
@@ -0,0 +1,75 @@
+CREATE INDEX pages_index ON pages
```

Comment:
On an empty database, it complain about `pg_search` not being installed. Doesn't hurt to add this:
```suggestion
CREATE EXTENSION IF NOT EXSITS pg_search;
CREATE INDEX pages_index ON pages
```

Reply:
Unfortunately, the `index` statements are actually interpreted currently, and their creation time is reported individually. Doing that would require a refactor, so will skip for now.